nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/collections/GetCollectionsTest.java

```diff
@@ -77,26 +79,23 @@ public void testGetCollections() throws Exception {
 
         PaginableDocumentModelListImpl collectionsList = (PaginableDocumentModelListImpl) service.run(ctx, chain);
 
-        assertNotNull(collectionsList);
         assertEquals(1, collectionsList.size());
         DocumentModel collection = collectionsList.get(0);
         assertEquals(COLLECTION_NAME + 1, collection.getName());
         assertEquals(COLLECTION_DESCRIPTION, collection.getPropertyValue("dc:description"));
 
         // Search several collections
         chain = new OperationChain("test-chain-2");
-        chain.add(GetCollectionsOperation.ID).set("searchTerm", COLLECTION_NAME + "%");
+        chain.add(GetCollectionsOperation.ID).set("searchTerm", COLLECTION_NAME.subSequence(0, 6));
```

Comment:
Are you removing a part of `COLLECTION_NAME`? If yes, what do you think about:
```suggestion
        chain.add(GetCollectionsOperation.ID).set("searchTerm", COLLECTION_NAME.subSequence(0, COLLECTION_NAME.size() - 3));
```

Reply:
Yes  thanks